src/app/_components/Header/Navigation/Navigation.tsx

```diff
@@ -0,0 +1,96 @@
+'use client';
+
+import Image from "next/image";
+import {useCallback, useEffect, useMemo, useState} from "react";
+import {usePathname} from "next/navigation";
+import {Menu} from "@deemlol/next-icons";
+import {NavigationItem, type NavigationItemProps} from "./NavigationItem";
+import {NavigationLabel} from "./NavigationLabel";
+
+const navigationItems: NavigationItemProps[] = [
+    {
+        href: "/",
+        label: "Start"
+    },
+    {
+        href: "/tajemniczy",
+        label: "Tajemniczy"
+    },
```

Comment:
@MalkizH3 @Klaym3n dochodzę do wniosku, że jednak fajniej będzie mieć tę podstronkę w menu. Pytanie jak chcemy to wyświetlać?